src/order/order_actions.py

```diff
@@ -23,9 +24,8 @@ def create(
         home_office_code: str,
         order_status_id: int = None,
         order_status: OrderModel = None,
-        uuid_param: str = None,
     ):
-        theUuid = uuid_param or str(uuid.uuid4())
+        theUuid = str(uuid.uuid4())
```

Comment:
The original is correct, somehow you lost the change

Reply:
I'm not sure how that happened. Those lines did not contain the conflict.